api/internal/git/repospec.go

```diff
@@ -27,9 +27,12 @@ type RepoSpec struct {
 	// TODO(monopole): Drop raw, use processed fields instead.
 	raw string
 
-	// Host, e.g. github.com
+	// Host, e.g. https://github.com/
```

Comment:
So I didn't change `Host` in any way. I just updated the comment because that is currently what the `Host` field holds and it was confusing that the comment only documented the domain. 

It might be a little tedious to add `Scheme`, because currently the `Host` includes, in addition to scheme, userinfo, and port. There also may be other components I don't know about. I'm happy to make the change, but it will take me 3 days.

Reply:
I moved `parseDomain` into `locloader.go`, as you recommended.